doc/design/model_market.md

```diff
@@ -0,0 +1,142 @@
+# Model Market
+
+The model market is a web site where model developers can publish and share their models and analysts can find some useful models to finish the analysis work. The model market can be deployed anywhere like on the cloud or on-premise with some configurations.
+
+In the [model zoo design](model_zoo.md), we described how model developers develop, publish and share custom models on SQLFlow, and how analysts can make use of the shared model with SQLFlow.
+
+Here, we'll describe how to build the model market model developers and analysts.
+
+## Overview
+
+The model market is designed to:
+
+1. Manage user login and logout.
+1. View published images available for the current user.
```

Comment:
What is this image?  model definition image?
Also add some explanation about the model definition image.

Reply:
Updated to "model definition Docker images". The details are explained in `model_zoo.md`